sample/README.md

```
@@ -12,41 +12,46 @@ Key parts of the code are annotated with `SETUP NOTE` comments. Refer to the fol
 - [build.gradle.kts](./build.gradle.kts) for installation, see `SETUP NOTE` comments.
 - [SampleApplication.kt](./src/main/java/com/klaviyo/sample/SampleApplication.kt) for initializing the Klaviyo SDK.
 - `SampleActivity.kt` for sample code to create/modify a profile, track events, and integrate push. This file
-  lives per product flavor — [manual](./src/manual/java/com/klaviyo/sample/SampleActivity.kt) and
-  [automatic](./src/automatic/java/com/klaviyo/sample/SampleActivity.kt) — see [Integration styles](#integration-styles-product-flavors) below.
+  lives per product flavor — [manual](./src/manual/java/com/klaviyo/sample/SampleActivity.kt),
+  [automatic](./src/automatic/java/com/klaviyo/sample/SampleActivity.kt), and
+  [unset](./src/unset/java/com/klaviyo/sample/SampleActivity.kt) — see [Integration styles](#integration-styles-product-flavors) below.
 - [Manifest](./src/main/AndroidManifest.xml) for push integration and other configurable settings.
 
 ## Integration styles (product flavors)
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)